src/assets/Icons/ArrowLeftIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { useTheme } from '../../theme/ThemeContext';
+import { colors } from '../../theme/colors';
+import { IconProps } from './IconProps';
+
+export const ArrowLeftIcon = ({size}:IconProps) => {
```

Comment:
Sugiero que el tamaño de los iconos se limiten a un cierto set de números. Veo que la propiedad es de tipo número, pero no creo que deberíamos dejar que un ícono se le de un tamaño de 100px, 5px o inclusive 500px; bueno, el número puede ser x, el cual hace que los tamaños entre iconos pierdan consistencia

https://github.com/AlanPinhon/deep-feels-ui/blob/icons-and-spacing/src/assets/Icons/IconProps.ts#L2

Tal vez puedes tomar como referencia este código:
https://github.com/bmcmahen/sancho/blob/master/src/Theme/index.ts#L67

Sólo que en vez de usar`px` deberíamos usar `rem`. Lo dejo a tu criterio 😄

Reply:
Tomé como referencia el código de sancho para utilizar tamaños limitados y ya están convertidos a `rem`
De todos modos cualquier comentario quedo pendiente. 🤓